pkg/lib/machine-dialogs.js

```diff
@@ -74,7 +75,8 @@
         "invalid-hostkey" : translate_and_init(invalid_hostkey_tmpl),
         "not-supported" : translate_and_init(not_supported_tmpl),
         "sync-users" : translate_and_init(sync_users_tmpl),
-        "unknown-hostkey" : translate_and_init(unknown_hosts_tmpl)
+        "unknown-hostkey" : translate_and_init(unknown_hosts_tmpl),
+        "unknown-host" : translate_and_init(unknown_hosts_tmpl)
```

Comment:
The difference is that if you get unknown-hostkey you should expect there to be a host-key field in the response json. We know that because we had permission to talk to the server and get it. With unknown-host we don't know the host-key because we don't have permission to talk to the server. To get that permission it needs to be a private channel or have the unknown-host param.